admin_solutions/master_node_config.adoc

```diff
@@ -130,12 +144,59 @@ $ ansible-playbook -b -i ./hosts ~/src/openshift-ansible/playbooks/byo/config.ym
 +
 The playbook updates the configuration, and restarts the {product-title} master service to apply the changes.
 
-You have now modified the master and node configuration files using Ansible,
-but this is just a simple use case. From here you can see which
+You have now modified the master and node configuration files using Ansible, but this is just a simple use case. From here you can see which
 xref:../admin_solutions/master_node_config.adoc#master-config-options[master] and
 xref:../admin_solutions/master_node_config.adoc#node-config-options[node configuration] options are
 https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.ose.example[exposed to Ansible] and customize your own Ansible inventory.
 
+[[htpasswd]]
+==== Using the `htpasswd` commmand
+
+To configure the {product-title} cluster to use HTPasswd authentication, you need at least one user with a hashed password. Add the username and password directly in the *_./hosts_* inventory file or pass the credentials in a flat file, as xref:config-htpasswd[as shown above]. 
+
+To create a user and hashed password
```

Comment:
Maybe format this as a procedure. 

To create a user and hashed password:

. Run:
+
----
$ htpasswd -n <user_name>
----

.  Enter and confirm . . .